webpack.config.js

```diff
@@ -92,7 +92,7 @@ const createEntrypoints = () => {
 
 		return {
 			...entries,
-			[ blockName ]: scriptPath,
+			[ blockName ]: `./packages/block-library/build-module/${ blockName }/frontend.js`,
```

Comment:
@gziolo, @vcanales: Before this change, the `import ... from './utils';` in `frontend.js` would cause `block-library/src/file/utils.js` to be imported when building the final blob, instead of `block-library/build-module/file/utils.js`.

This appears to be the smallest change I could make to fix this issue, I'd appreciate your testing and feedback on it.

Reply:
Actually, I think you should be able to update the same path in line 83 instead. I now realized that we should never scan `src` folder because webpack in Gutenberg operates on packages as if they were published to npm. That means in practice that webpack is only concerned about `build-module` folders in WordPress packages. I think it's a good change. @youknowriad, can you double check?

Reply:
Sounds good to me, I was not aware of that distinction between `src` and `build-module`.

A final edit, would mean:

- changing L72 to

```js
const blockNameRegex = new RegExp( /(?<=build-module\/).*(?=(\/frontend))/g );
```

- L82-84 to

```js
const scriptPaths = fastGlob.sync(
	'./packages/block-library/build-module/**/frontend.js'
);
```

- Reverting this
```suggestion
			[ blockName ]: scriptPath,
```

I've tested these changes on https://github.com/WordPress/gutenberg/pull/30047, and everything works as expected.

Reply:
👍🏻 ecb14079a1